File: app/src/main/java/com/example/myers/diff/Snake.java
package com.example.myers.diff;

/**
 * @author gk
 * @Description:
 * @date 2023/12/25
 */
public final class Snake extends PathNode {
    public Snake(int i, int j, PathNode prev) {
        super(i, j, prev);
    }

    @Override
    public Boolean isSnake() {
        return true;
    }
}

File: app/src/main/java/com/example/myers/diff/Snake.kt
package com.example.myers.diff

/**
 * @author gk
 * @Description:
 * @date 2023/12/25
 */
class Snake(i: Int, j: Int, prev: PathNode?) : PathNode(i, j, prev) {
    override val isSnake: Boolean
        get() = true
}